client/src/components/AddNewTaskBtn/index.js

```diff
@@ -0,0 +1,19 @@
+import styles from './style.module.css'
+
+const AddNewTaskBtn = () => {
+
+    const handleClick = () => {
+        console.log('Add New Task Button clicked');
```

Comment:
Just curious, leaving this console here was intentional?

Reply:
Yes, I was just testing my functionality. It would be removed once there's a form to control from the onClick.